### PR TITLE
changes made for swymed

### PR DESCRIPTION
**Pull request by Vidyaaa24**

### CODE REVIEW

File: src/main/java/com/iemr/tm/controller/videoconsultation/VideoConsultationController.java [hunk 1]
```diff
@@ -97,20 +97,20 @@ public String call(@PathVariable("fromuserID") Long fromuserID, @PathVariable("t
 	}
 
 	@CrossOrigin()
-	@ApiOperation(value = "Swymed service for users by passing type", consumes = "application/json", produces = "application/json")	
+	@ApiOperation(value = "Video consultation service for users by passing type", consumes = "application/json", produces = "application/json")	
 	@RequestMapping(value = "/call/{fromuserID}/{touserID}/{type}", headers = "Authorization", method = {
 			RequestMethod.GET }, produces = { "application/json" })
-	public String CallSwymedAndJitsi(@PathVariable("fromuserID") Long fromuserID,
+	public String CallVideoConsultationAndJitsi(@PathVariable("fromuserID") Long fromuserID,
```
Comment (harimohanr) [inline src/main/java/com/iemr/tm/controller/videoconsultation/VideoConsultationController.java:103]: @Vidyaaa24 Method name starts with capital letter. please correct it

File: src/main/java/com/iemr/tm/controller/videoconsultation/VideoConsultationController.java [hunk 2]
```diff
@@ -154,21 +154,21 @@ public String callvan(@PathVariable("fromuserID") Long fromuserID, @PathVariable
 	}
 
 	@CrossOrigin()
-	@ApiOperation(value = "Call van through Swymed by passing type", consumes = "application/json", produces = "application/json")
+	@ApiOperation(value = "Call van through video consultation by passing type", consumes = "application/json", produces = "application/json")
 	@RequestMapping(value = "/callvan/{fromuserID}/{vanID}/{type}", headers = "Authorization", method = {
 			RequestMethod.GET }, produces = { "application/json" })
-	public String CallVanSwymedAndJitsi(@PathVariable("fromuserID") Long fromuserID,
+	public String CallVanVideoConsultationAndJitsi(@PathVariable("fromuserID") Long fromuserID,
```
Comment (harimohanr) [inline src/main/java/com/iemr/tm/controller/videoconsultation/VideoConsultationController.java:160]: @Vidyaaa24 Method name starts with capital letter

**Review by harimohanr — CHANGES_REQUESTED** [review]
@Vidyaaa24 Fix the review comments and resubmit

File: src/main/java/com/iemr/tm/controller/videoconsultation/VideoConsultationController.java [hunk 3]
```diff
@@ -19,7 +19,7 @@
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see https://www.gnu.org/licenses/.
 */
-package com.iemr.tm.controller.swymedcontroller;
+package com.iemr.tm.controller.videoconsultation;
```
Comment (Anjana2310) [inline src/main/java/com/iemr/tm/controller/videoconsultation/VideoConsultationController.java:22]: controller word is missing

File: src/main/java/com/iemr/tm/service/videoconsultation/VideoConsultationServiceImpl.java [hunk 4]
```diff
@@ -19,103 +19,99 @@
...
-import com.iemr.tm.utils.exception.SwymedException;
+import com.iemr.tm.utils.exception.VideoConsultationException;
 
 @Service
-public class SwymedServiceImpl implements SwymedService {
+public class VideoConsultationServiceImpl implements VideoConsultationService {
 
-	// swymed://dnsname?l=mylogin&p=mypassword&d=mydomain&c=callnumber&m=1
-	private String swymed_dnsname = ConfigProperties.getPropertyByName("swymed_dnsname");
+	private String videoConsultationDNSName = ConfigProperties.getPropertyByName("videoConsultationDNSName");
```
Comment (Anjana2310) [inline src/main/java/com/iemr/tm/service/videoconsultation/VideoConsultationServiceImpl.java:41]: ConfigProperties.getPropertyByName("swymed_dnsname");
swymed_dnsname is a property mentioned in application.properties.

File: src/main/java/com/iemr/tm/controller/videoconsultationcontroller/VideoConsultationController.java [hunk 5]
```diff
@@ -154,21 +154,21 @@ public String callvan(@PathVariable("fromuserID") Long fromuserID, @PathVariable
...
 			@PathVariable("vanID") Integer vanID, @PathVariable("type") String Type) {
 
 		OutputResponse response = new OutputResponse();
 
 		try {
 
 			String createdData = null;
-			if (Type.equalsIgnoreCase("Swymed")) {
-				createdData = swymedService.callVan(fromuserID, vanID);
+			if (Type.equalsIgnoreCase("VideoConsultation")) {
```
Comment (Anjana2310) [inline src/main/java/com/iemr/tm/controller/videoconsultationcontroller/VideoConsultationController.java:168]: if (Type.equalsIgnoreCase("Swymed"))

File: src/main/java/com/iemr/tm/controller/videoconsultationcontroller/VideoConsultationController.java [hunk 6]
```diff
@@ -154,21 +154,21 @@ public String callvan(@PathVariable("fromuserID") Long fromuserID, @PathVariable
 	}
 
 	@CrossOrigin()
-	@ApiOperation(value = "Call van through Swymed by passing type", consumes = "application/json", produces = "application/json")
+	@ApiOperation(value = "Call van through video consultation by passing type", consumes = "application/json", produces = "application/json")
 	@RequestMapping(value = "/callvan/{fromuserID}/{vanID}/{type}", headers = "Authorization", method = {
 			RequestMethod.GET }, produces = { "application/json" })
-	public String CallVanSwymedAndJitsi(@PathVariable("fromuserID") Long fromuserID,
+	public String callVanVideoConsultationAndJitsi(@PathVariable("fromuserID") Long fromuserID,
```
Comment (Anjana2310) [inline src/main/java/com/iemr/tm/controller/videoconsultationcontroller/VideoConsultationController.java:160]: We will maintain the previous name

**Review by Anjana2310 — CHANGES_REQUESTED** [review]
@Vidyaaa24 Pls fix the review comments

**Review by harimohanr — APPROVED** [review]
@looks oka to me

**Pull request merged**